### PR TITLE
Move to pillow

### PR DESCRIPTION
**Pull request by decobots**
src/deltas_research.py will not be used in the future

### CODE REVIEW

File: pipenv [hunk 1]
```diff
@@ -0,0 +1,101 @@
+��� ����祭�� ᢥ����� �� ��।������� ������� ������ HELP <��� �������>
```
Comment (livercat) [inline pipenv]: ???

File: main.py [hunk 2]
```diff
@@ -16,10 +17,13 @@
         table = PhotoTable(db=db, table_name="main_photos")
         twitter = Twitter(table=table)
         flickr = Flickr(table=table)
+        color_table = ColorTable()
         for _ in range(will_be_uploaded_N_photos):
             pictures = flickr.get_photos()
             photo = flickr.random_photo(pictures)
             photo = flickr.get_photo(photo)
+            average_color_of_photo = ImageAnalazer(photo.data).avg_color
+            neariest_color = color_table.nearest_color(average_color_of_photo.lab)
```
Comment (livercat) [inline main.py]: Typo: nearest

File: src/image_analyzer.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import io
+
+from PIL import Image
+from PIL import ImageStat
```
Comment (livercat) [inline src/image_analyzer.py]: You can move this import to the previous line

File: src/svgmaker.py [hunk 4]
```diff
@@ -4,31 +4,38 @@
...
+from src.color_table import Color, ColorTable, colors_delta
 
 DATA_DEFAULT_PATH = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'data')
 W = 140
 H = 40
 
 
-def create_svg(color_groups: List[List['Color']]):
-    with open(os.path.join(DATA_DEFAULT_PATH, "test.svg"), mode='w') as f:
+def create_svg(color_groups: List[List['colors_delta']], filename: str):
```
Comment (livercat) [inline src/svgmaker.py]: 'colors_delta' -> colors_delta

**Review by livercat — CHANGES_REQUESTED** [review]

File: main.py (hunk 2 above)
Comment (decobots) [reply main.py]: Thanks!

File: pipenv (hunk 1 above)
Comment (decobots) [reply pipenv]: deleted

File: src/image_analyzer.py (hunk 3 above)
Comment (decobots) [reply src/image_analyzer.py]: done

File: src/svgmaker.py (hunk 4 above)
Comment (decobots) [reply src/svgmaker.py]: done

**Review by livercat — APPROVED** [review]

**Pull request merged**